src/components/common/SidebarDesktop.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+    <!-- Static sidebar for desktop -->
```

Comment:
Similar to the header component, please make the sidebar component responsive rather than interchanging multiple components depending on the view